packages/front-end/components/Features/RuleModal.tsx

```diff
@@ -152,6 +155,13 @@ export default function RuleModal({
   const experimentId = form.watch("experimentId");
   const selectedExperiment = experimentsMap.get(experimentId) || null;
 
+  const { data: sdkConnectionsData } = useSDKConnections();
+  const hasSDKWithNoBucketingV2 = !getConnectionsSDKCapabilities({
+    connections: sdkConnectionsData?.connections ?? [],
+    mustMatchAllConnections: true,
+    project: feature.project,
+  }).includes("bucketingV2");
```

Comment:
This same code is copy/pasted 3 times.  Can we abstract this?

Reply:
sure